setup.py

```diff
@@ -61,7 +61,7 @@
 
 # Work around clang raising hard error for unused arguments
 if sys.platform == "darwin":
-    os.environ['CFLAGS'] = "-Qunused-arguments"
+    os.environ['CFLAGS'] += " -Qunused-arguments -I/opt/homebrew/opt/sqlcipher/include -L/opt/homebrew/opt/sqlcipher/lib -L/opt/homebrew/opt/openssl/lib/ -lcrypto"
```

Comment:
Does this break users that are not using MacOS, such as Linux?

Reply:
no, because it is under darwin